### PR TITLE
HashLink compatibility

### PR DESCRIPTION
**Pull request by libauror-a**
fuck neko, embrasse hashlink, just some problems, NGio dosen't work because you'll get this error `FATAL ERROR : Failed to load library ssl.hdll` so i had to jankly fix the error, (probally modders shoudn't use the NG shit in their mods/engines and shoud be coded out) probally gimme suggestions because this is kida bad, but it still works!

### CODE REVIEW

File: source/Discord.hx [hunk 1]
```diff
@@ -84,4 +85,9 @@ class DiscordClient
 
 		//trace('Discord RPC Updated. Arguments: $details, $state, $smallImageKey, $hasStartTimestamp, $endTimestamp');
 	}
+	#else
+	public function new() {
+		trace("Discord Client isn't going to start because this is a cpp client, go home.");
```
Comment (ACrazyTown) [inline source/Discord.hx:90]: looks good, but shouldn't it be "because this is *not* a cpp client"?

**Review by ACrazyTown — CHANGES_REQUESTED** [review]

**Comment by DotWith** [comment]
Also do the same with newgrounds

**Comment by ThePlank** [comment]
> OpenGLRenderer crash

i think lime and openfl are outdated

**Comment by DotWith** [comment]
> > OpenGLRenderer crash
> 
> i think lime and openfl are outdated

Thanks I was installing a dev openfl from git. It now works with non git version.

**Comment by KitOdin** [comment]
since when was there a secret repo for fnf?

**Comment by Steviegt6** [comment]
> since when was there a secret repo for fnf?

For quite some time, how else would they collaboratively develop it?

**Pull request closed without merging**